response_operations_ui/templates/reporting-unit.html

```diff
@@ -148,7 +148,7 @@ <h4 class="neptune">Respondents</h4>
                 <dd>
                 Verification email sent
                 <br>
-                  <a href="{{ url_for('reporting_unit_bp.resend_verification', ru_ref=ru_ref, party_id=party_id) }}">Re-send verification email</a>
+                  <a href="{{ url_for('reporting_unit_bp.resend_verification', ru_ref=ru_ref, party_id=respondent.id) }}">Re-send verification email</a>
```

Comment:
reporting_unit_bp.view_resend_verification?

The user journey here should be when the user clicks on Re-send verification email they are directed to view resend verification page (this is a speed bump so the user does not accidentally click the resend button). 

You can follow the way it is being done [here](https://github.com/ONSdigital/response-operations-ui/blob/f055314591a205eaed8e92994ffe96e9b8d1d3ad/response_operations_ui/templates/reporting-unit.html#L184).

Reply:
In what scenario are you seeing this?  For me, it always redirects back to the previous page as it should.

Reply:
So it shouldn't be doing that (redirects back to the previous page). It should take you to a confirmation page which is rendered in the reporting_unit_bp.view_resend_verification function. Once you've confirmed it should redirect to the previous page.